### ISSUE TITLE
Temas sustantivos

### ISSUE THREAD

**Issue by juancarloscastillo** [justicia-distributiva/pref-redis-la-colmex#2]
(re) Leyendo ahora sobre preferencias, hay un par de temas que aparecen recurrentemente, sobre todo en literatura de comparación internacional: corrupción y confianza en instituciones. ¿Les parece también incorporar algo de esos temas?

**Comment by gonzalofranetovic** [comment]
Sí, de acuerdo. Más aún, considerando que en nuestro paper la confianza en instituciones era uno de los factores más relevantes en el apoyo a redistribución en AL.

**Comment by juancarloscastillo** [comment]
OK, @jciturras ? @Kevin-carrasco ? qué les parece? Tal vez podríamos reorganizar incluyendo estos temas: una sección más bien socio-estructural (ingreso, desigualdad, etc) y una institucional (confianza-corrupción?

**Comment by jciturras** [comment]
Hola, me parece buena idea y así también tiene un plus que lo diferencia del paper original. Podría ser un indicador de corrupción (Quality of Government) o corrupción percibida (Corruption Perceptions Index). 

Acá se pueden explorar: https://lines-plot.qog.gu.se/
